tests/tests/poa.rs

```diff
@@ -85,6 +90,74 @@ async fn can_get_sealed_block_from_poa_produced_block() {
         .expect("failed to verify signature");
 }
 
+#[tokio::test]
+async fn can_get_sealed_block_from_poa_produced_block_when_signing_with_kms() {
+    // This test is only enabled if the environment variable is set
+    let Some(kms_arn) = option_env!("FUEL_CORE_TEST_AWS_KMS_ARN") else {
+        panic!("KMS not enabled");
+        // return;
```

Comment:
This panic will be changed to the return when I have made made sure that the test actually fails without KMS.

Reply:
removed in 4f14fda77